clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -3408,6 +3408,8 @@ def warn_objc_redundant_literal_use : Warning<
 def err_attr_tlsmodel_arg : Error<"tls_model must be \"global-dynamic\", "
   "\"local-dynamic\", \"initial-exec\" or \"local-exec\"">;
 
+def err_attr_codemodel_arg : Error<"code_model '%0' is not yet supported on this target">;
```

Comment:
We should remove `yet`, because a target may decide not to support the attribute.

The AIX err below is for progress tracking.

Reply:
Done. Thanks